src/main/java/redis/clients/jedis/commands/KeyCommands.java

```diff
@@ -119,6 +120,31 @@
    */
   long expire(String key, long seconds);
 
+  /**
+   * Similar to {@link KeyCommands#expire(String, long) EXPIRE} but with optional expiry setting.
+   * @see KeyCommands#expire(String, long)
+   * @param key
+   * @param seconds time to expire
+   * @param setExpireOption can be NX, XX, GT or LT
+   * @return 1 if the timeout was set, 0 otherwise. Since the key already has an associated timeout
+   * (this may happen only in Redis versions &lt; 2.1.3, Redis &gt;= 2.1.3 will happily update the timeout),
+   * or the key does not exist.
+   */
+  long expire(String key, long seconds, SetExpireOption setExpireOption);
+
+  /**
+   * <b><a href="http://redis.io/commands/expiretime">ExpireTime Command</a></b>
+   * Returns the absolute Unix timestamp (since January 1, 1970) in seconds at which the given key will expire.
+   * <p>
+   * The command returns -1 if the key exists but has no associated expiration time, and -2 if the key does not exist.
+   * <p>
+   * Time complexity: O(1)
+   * @param key
+   * @return Expiration Unix timestamp in seconds, or a negative value in order to signal an error:
+   * -1 if the key exists but has no associated expiration time, and -2 if the key does not exist.
+   */
+  long expiretime(String key);
```

Comment:
[DISCUSSION]
@dengliming @yangbodong22011 Should we name `expireTime`?

Reply:
Both are fine to me : ). Because we have a lot of places that don't follow the hump naming.

Reply:
@dengliming That's mostly historically. But as Redis commands are getting (somewhat) longer where we can confidently identify multiple complete words, IMO it's time we gradually adopt humping.

@yangbodong22011